CHANGELOG.md

```diff
@@ -42,6 +42,7 @@
 * Add Item Limit to Loan Policy. Refs UICIRC-390.
 * Overdue fine policy: Validation field incorrect for overdue recall fine. Refs UIU-1232.
 * Lost item fee policy: Words "late" and "Set cost" missing on entry of new policy. Refs UIU-1315.
+* Remove unnecessary data from `<LoanHistory>` and add tests for anonymizing intervals. Refs UIU-1042.
```

Comment:
> Remove unnecessary data from `<LoanHistory>`

This is not quite unclear what exactly you have removed, whether unnecessary methods from the component or something else. Please make the message more clear.